src/main/java/meshIneBits/Mesh.java

```diff
@@ -333,10 +337,37 @@ public void update(Observable o, Object arg) {
                     break;
                 case GLUED:
                     break;
+                case ORDENNING:
```

Comment:
You mean "ORDERRING"?

Reply:
Indeed and should be better to use SCHEDULING / SCHEDULED instead

Reply:
+1 SCHEDULING / SCHEDULED is actually more intuitive

Reply:
So please change into SCHEDULING / SCHEDULED and commit it 👍